SwiftCoAP_Library/SCServer.swift

```diff
@@ -57,7 +57,7 @@ class SCServer: NSObject {
     fileprivate class SCAddressWrapper: NSObject {
         let hostname: String
         let port: UInt16
-        override var hashValue: Int {
+        override var hash: Int {
```

Comment:
same as above

Reply:
Xcodee Warning: 
```Override of 'NSObject.hashValue' is deprecated; override 'NSObject.hash' to get consistent hashing behavior```

hashValue changed from swift 3 --> hash in swift 4 ...


![](https://i.ibb.co/Pty2FBm/Screenshot-2019-12-24-at-02-20-33.png)